components/dashboard/src/menu/OrganizationSelector.tsx

```diff
@@ -110,12 +100,22 @@ export default function OrganizationSelector() {
             // collaborator can read org setting via API so that other feature like restrict org workspace classes could work
             // we only hide the menu from dashboard
             linkEntries.push({
-                title: "Settings",
-                customContent: <LinkEntry>Settings</LinkEntry>,
+                title: "Organization Settings",
+                customContent: <LinkEntry>Organization Settings</LinkEntry>,
```

Comment:
**thought**: I'd keep that as it, and instead try to merge these with the user settings, see UX-37.

```suggestion
                title: "Settings",
                customContent: <LinkEntry>Settings</LinkEntry>,
```

Reply:
We decided to do this as to reduce confusion with the 2 "settings" entries. I think it will also help to clear out the confusion between user and organization settings, because the org switcher never actually discloses it's for switching organizations.

Reply:
There's definitely confusion around settings. ✔️ 

I'd take a closer look at the UX-37 linked above, and maybe discuss with the team if this UX proposal makes sense. Happy to chat more about it. Cc @corygrunk

Reply:
Thanks a lot for sharing that, I'd be really happy about us unifying the two interfaces. Happy to join in when we do